package.json

```diff
@@ -3,67 +3,68 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "fix": "preconstruct fix",
-    "dev": "preconstruct dev",
-    "build": "preconstruct build",
-    "watch": "preconstruct watch",
-    "fixtures": "pnpm --recursive --filter='@crackle-fixtures/*'",
-    "format": "npm-run-all format:eslint format:prettier",
-    "format:eslint": "eslint --fix --cache .",
-    "format:prettier": "prettier --write --cache .",
-    "lint": "npm-run-all --parallel --aggregate-output --continue-on-error 'lint:*'",
+    "%packages": "pnpm --filter='@crackle/*'",
+    "%fixtures": "pnpm --filter='@crackle-fixtures/*' --aggregate-output",
+    "bootstrap": "pnpm %packages exec crackle-bootstrap package --fix",
+    "dev": "pnpm --aggregate-output %packages dev",
+    "build": "pnpm --aggregate-output %packages build",
+    "build:ci": "wireit",
+    "format": "wireit",
+    "format:eslint": "eslint --cache --fix .",
+    "format:prettier": "wireit",
+    "lint": "wireit",
     "lint:eslint": "eslint --cache .",
-    "lint:prettier": "prettier --cache --check .",
-    "lint:tsc": "tsc --noEmit",
-    "test": "pnpm test:vitest --run && pnpm test:playwright",
-    "test:vitest": "vitest",
-    "test:playwright": "playwright test",
-    "add-fixture": "tsx ./scripts/add-fixture.ts",
-    "sync-dependencies": "pnpm --recursive sync-dependencies && pnpm install",
-    "prepare-publish": "pnpm build",
+    "lint:prettier": "wireit",
+    "lint:tsc": "wireit",
+    "test": "pnpm test:unit:ci && pnpm test:integration:ci && pnpm test:playwright",
+    "test:unit": "vitest",
+    "test:unit:ci": "pnpm test:unit --run",
+    "test:integration": "vitest --config vitest.config.integration.ts",
+    "test:integration:ci": "pnpm test:integration --run",
+    "test:playwright": "wireit",
+    "fixtures:add": "tsx ./scripts/add-fixture.ts",
+    "fixtures:dev": "pnpm %fixtures dev",
+    "fixtures:package": "pnpm %fixtures package",
+    "sync:dependencies": "pnpm --recursive sync:dependencies && pnpm install",
+    "sync:prettierignore": "ignore-sync .prettierignore-sync",
+    "prepare-publish": "pnpm build:ci",
     "release": "pnpm prepare-publish && changeset publish",
     "version": "changeset version && pnpm install --lockfile-only"
   },
   "dependencies": {
-    "@babel/core": "^7.19.4",
-    "@babel/preset-env": "^7.19.4",
+    "@babel/core": "^7.20.7",
+    "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
     "@changesets/changelog-github": "^0.4.7",
     "@changesets/cli": "^2.25.0",
+    "@crackle-private/bootstrap": "workspace:*",
     "@crackle/cli": "workspace:*",
-    "@playwright/test": "1.22.2",
-    "@preconstruct/cli": "^2.2.2",
+    "@crackle/core": "workspace:*",
+    "@playwright/test": "^1.32.0",
     "@preconstruct/eslint-plugin-format-js-tag": "^0.2.0",
-    "@types/fs-extra": "^9.0.13",
+    "@types/fs-extra": "^11.0.1",
```

Comment:
Should this be v10 to match the [package version]?

[package version]: https://github.com/seek-oss/crackle/pull/84/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R53